src/Node/FS.purs

```diff
@@ -7,6 +17,32 @@ import Prelude
 --
 foreign import data FS :: !
 
+foreign import data FileDescriptor :: *
+
+data FileFlags = R | R_PLUS | RS | RS_PLUS
+               | W | WX | W_PLUS | WX_PLUS
+               | A | AX | A_PLUS | AX_PLUS
+
+instance showFileFlags :: Show FileFlags where
```

Comment:
I agree that having Show's output be evaluatable is nice but not essential. More important, in my view, is not overloading `show` with lots of different meanings, because this can be unsafe. Conceptually, the action we're performing on a `FileFlags` value when calling out to the FFI is "serialize this FileFlags value into a string according to the format understood by Node.js' filesystem api". This is not what `Show` is for - this concept isn't really generalizable to other types, as far as I can tell - and so using an abstraction like `Show` for this concept is essentially a lie.

I won't ask you to write out a proper `Show` instance for `FileFlags` (although that would be nice), but I am fairly certain in my belief that this code should be monomorphic.

Also, perhaps it's not particularly important to note, but there is no `read` in PureScript. Not yet, at least.

> The existing purescript-node-fs codebase doesn't do this

Thanks for pointing that out - I will change other places where `Show` is misused, then.